pkg/scheduler/sorters_test.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2019 Cloudera, Inc.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import (
+	"github.com/cloudera/yunikorn-core/pkg/common/resources"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// verify queue ordering is working
+func TestSortQueues(t *testing.T) {
+	root, err := createRootQueue()
+	root.CachedQueueInfo.GuaranteedResource = resources.NewResourceFromMap(
+		map[string]resources.Quantity{"memory": 1000, "vcore": 1000})
+	if err != nil {
```

Comment:
should swap the err check and the use of the returned queue as it could be nil (throughout the file)